code/modules/antagonists/cult/cult_items.dm

```diff
@@ -83,6 +84,7 @@
 	name = "bloody bastard sword"
 	desc = "An enormous sword used by Nar'Sien cultists to rapidly harvest the souls of non-believers."
 	w_class = WEIGHT_CLASS_HUGE
+	total_mass = 3.9
```

Comment:
A cult sword that has 1 hit kills that removes people from the round should take some more stam then just 3.9 mass.
JUST HOLDING IT is 50% block on *all attacks* + a mirror sheild mask you a walking tank that is able to ware the flatulent robes with easy